Provision/base.sql

```diff
@@ -57,21 +50,21 @@ CREATE TABLE IF NOT EXISTS summoners
     created_at DATETIME DEFAULT NOW() NOT NULL,
     updated_at DATETIME DEFAULT NOW() NOT NULL,
     PRIMARY KEY (id),
-    FOREIGN KEY (player_icon_id) REFERENCES player_icon(id),
     FOREIGN KEY (users_id) REFERENCES users(id),
     FOREIGN KEY (division_ranks_id) REFERENCES division_ranks(id)
 );
 
 CREATE TABLE IF NOT EXISTS matches
 (
     id INT NOT NULL AUTO_INCREMENT,
-    role ENUM('Top', 'Jungle', 'Middle', 'Bottom', 'Support') not null,
+    match_id VARCHAR(255) NOT NULL,
+    lane ENUM('Top', 'Jungle', 'Mid', 'Bottom', 'Support') not null,
+    role VARCHAR(45) NULL,
```

Comment:
Lets bump this to 255, no real performance hit and we don't have a guarantee of what length of data will come out of the riot api in the future.